.github/workflows/nightly-testing.yaml

```diff
@@ -0,0 +1,11 @@
+name: Nightly Testing
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Runs at midnight every day
```

Comment:
This is technically UTC time, so 6pm mountain. I think that's OK but we could probably move it a few hours later (6 UTC?) so that its outside the timeframe of our normal work/after all changes for a day.